include/arch/arm/arch/machine/gic_v2.h

```diff
@@ -17,10 +17,11 @@
 #include <linker.h>
 #include <mode/smp/smp.h>
 #include <model/statedata.h>
-
+#include <mode/machine/gic_v2_gen.h>
```

Comment:
Should this be a condition include based on `CONFIG_ARM_HYPERVISOR_SUPPORT` (which would make it consistent with `gic_v3.h`

Reply:
Sure.